website/data/plugin-framework-nav-data.json

```diff
@@ -79,5 +82,111 @@
   {
     "title": "Debugging",
     "path": "debugging"
+  },
+  {
+    "title": "Migrating from SDK",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "migrating"
+      },
+      {
+        "title": "Testing",
+        "path": "migrating/testing"
+      },
+      {
+        "title": "Schema",
+        "routes": [
+          {
+            "title": "Overview",
+            "path": "migrating/schema"
+          }
+        ]
+      },
+      {
+        "title": "Providers",
+        "routes": [
+          {
+            "title": "Overview",
+            "path": "migrating/providers"
+          }
+        ]
+      },
+      {
+        "title": "Resources",
+        "routes": [
+          {
+            "title": "Overview",
+            "path": "migrating/resources"
+          },
+          {
+            "title": "CRUD Functions",
+            "path": "migrating/resources/crud"
+          },
+          {
+            "title": "Import",
+            "path": "migrating/resources/import"
+          },
+          {
+            "title": "Plan Modification",
+            "path": "migrating/resources/plan-modification"
+          },
+          {
+            "title": "State Upgraders",
+            "path": "migrating/resources/state-upgrade"
+          }
+        ]
+      },
+      {
+        "title": "Data Sources",
+        "routes": [
+          {
+            "title": "Overview",
+            "path": "migrating/data-sources"
+          }
+        ]
+      },
+      {
+        "title": "Attributes & Blocks",
+        "routes": [
+          {
+            "title": "Attribute Schema",
+            "path": "migrating/attributes-blocks/attribute-schema"
+          },
+          {
+            "title": "Attribute Types",
+            "path": "migrating/attributes-blocks/types"
+          },
+          {
+            "title": "Attribute Fields",
+            "path": "migrating/attributes-blocks/fields"
+          },
+          {
+            "title": "Default Values",
+            "path": "migrating/attributes-blocks/default-values"
+          },
+          {
+            "title": "Force New",
+            "path": "migrating/attributes-blocks/force-new"
+          },
+          {
+            "title": "Validators - Predefined",
+            "path": "migrating/attributes-blocks/validators-predefined"
+          },
+          {
+            "title": "Validators - Custom",
+            "path": "migrating/attributes-blocks/validators-custom"
+          },
+          {
+            "title": "Blocks",
+            "path": "migrating/attributes-blocks/blocks"
+          },
+          {
+            "title": "Blocks with Computed Fields",
+            "path": "migrating/attributes-blocks/blocks-computed"
+          }
+        ]
+      }
```

Comment:
Now that we have a Schema section, should these pages be moved under there as well?

Reply:
I think that makes sense given that each of these pages relates to how the schema is set-up.

@laurapacilio @robin-norwood are you happy if I go ahead and move these pages as @bflad suggests?